terraform/aws/main.tf

```diff
@@ -76,6 +76,16 @@ resource "aws_security_group_rule" "p2p-{{ name }}" {
   security_group_id = "${aws_security_group.main-{{ name }}.id}"
 }
 
+resource "aws_security_group_rule" "p2p-proxy-{{ name }}" {
```

Comment:
No, polkadot-secure-validator doesn't have yet the concept of a bastion (we will probably include it soon), so node exporter port is open to the public internet. We use an nginx proxy with basic auth enabled in front of node-exporter to prevent unwanted access.

Reply:
:+1: Ok. I guess basic auth is good enough in the short term, but I think long term this port should either not be publicly routable or authenticated via a stronger schema e.g. mutual TLS.